package.json

```diff
@@ -481,8 +481,7 @@
     "compile": "tsc -p . && webpack --config webpack.config.js",
     "watch": "webpack --config webpack.config.js --watch --info-verbosity verbose",
     "build": "webpack --config webpack.config.js --mode=\"production\"",
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "node ./scripts/download-vscode-for-system-tests && node ./scripts/install-vsix-dependencies redhat.java && node ./scripts/install-vsix-dependencies vscode-java-debug-0.20.0.vsix && node ./scripts/run-vscode-tests"
+    "test": "tsc -p . && node ./out/test/index.js"
```

Comment:
How to install the dependencies?

Reply:
+1

How to run it in cli?

Reply:
I'm following the [official guide](https://code.visualstudio.com/api/working-with-extensions/testing-extension#custom-setup-with-vscodetest) to install redhat.java before running the test cases. 
But as you can see from the CI log, the `vscodeExecutablePath` does NOT point to the code CLI as expected, thus cannot successfully install the extension.